cms/templates/partials/nhs_components/select_search_results_by_order.html

```diff
@@ -16,6 +15,9 @@ <h1 class="nhsuk-fieldset__heading">Sort by:</h1>
           <option value="-latest_revision_created_at"{% if order == '-latest_revision_created_at' %} selected{% endif %}>Most relevant</option>
           <option value="latest_revision_created_at"{% if order == 'latest_revision_created_at' %} selected{% endif %}>Newest first</option>
           <option value="-first_published_at"{% if order == '-first_published_at' %} selected{% endif %}>Oldest first</option>
+          <option value="most_relevant"{% if order == "most_relevant" %} selected{% endif %}>Most relevant</option>
```

Comment:
This is almost certainly a placeholder which I was working on just before I dropped it -- we probably need some code to make whatever this value is not sort the search results at all (and trust ElasticSearch returns them in a sensible order.)

This almost certainly currently breaks due to not being a valid search order.

My plan was to detect this value and not `.order_by()` the results if encountered in templates/search/views.py

Reply:
Can we raise this as a new backlog ticket, to fix ordering?

Reply:
Agreed! Can you link/copypaste that text into the ticket?